helm-service/chart/values.yaml

```diff
@@ -5,6 +5,7 @@ helmservice:
     tag: ""                                  # Container Tag
   service:
     enabled: true                            # Creates a Kubernetes Service for the helm-service
+  gracePeriod: 90                            # PreStop hook +30s
```

Comment:
is there a reason why the grace period is configurable but the prestop hook time isn't?